fenix/app/src/main/res/values/strings.xml

```diff
@@ -581,6 +581,17 @@
     <!-- Summary for notification preference indicating notifications are not allowed -->
     <string name="notifications_not_allowed_summary">Not allowed</string>
 
+    <!-- Add-on Permissions -->
+    <!-- The title of one the section with required permissions from addon's permissions screen -->
+    <string name="mozac_feature_addons_permissions_heading_required">Required</string>
```

Comment:
I think instead of removing the prefix they should actually be part of AC since these are the new MV3 requirements and should be available to any project that uses AC. I've moved them and update can be found [here](https://phabricator.services.mozilla.com/D205398)

Reply:
I reverted this change and removed the prefix, moving to AC is not the right approach since it's not part of the addon feature. That was just a mistake in thinking.